src/types/dto/workflowStates.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
+
+export const WorkflowStateTypeSchema = z.enum(['backlog', 'unstarted', 'started', 'completed', 'cancelled'])
```

Comment:
We may need the enum values later

```suggestion
export enum WorkflowState {
  BACKLOG = 'backlog',
  UNSTARTED = 'unstarted',
  STARTED = 'started',
  COMPLETED = 'completed',
  CANCELLED = 'cancelled'
}
export const WorkflowStateTypeSchema = z.nativeEnum(WorkflowState)
```